extensions/data-workspace/src/dialogs/newProjectDialog.ts

```diff
@@ -159,7 +176,26 @@ export class NewProjectDialog extends DialogBase {
 			this.model.location = selectedFolder;
 		}));
 
-		const form = view.modelBuilder.formContainer().withFormItems([
+		const targetPlatformDropdown = view.modelBuilder.dropDown().withProperties<azdataType.DropDownProperties>({
```

Comment:
FYI you can use `withProps` to get proper typing without having to specify the properties type yoursel